pkg/resources/api_authentication_integration_with_client_credentials.go

```diff
@@ -0,0 +1,486 @@
+package resources
```

Comment:
I have almost the same comments for this file.

I also don't like, that multiple implementations are just copied here and not extracted for each of the integration type (e.g. handleCommonProperties - common + handleAdditionalPropertiesForX - for every resource; and similarly for other places).

Reply:
Extracted.